rate_counter/rate_counter/settings.py

```diff
@@ -40,6 +41,15 @@
     'rate'
 ]
 
+# CELERY STUFF
+INSTALLED_APPS += ("djcelery", )
+BROKER_URL = 'redis://localhost:6379/0'
```

Comment:
Here you are explicitly using localhost. But when we will upload project on hosting I am not sure about it. You can make there a comment that this is setup for dev machines.